qt/widgets/instrumentview/src/BankRenderingHelpers.cpp

```diff
@@ -167,7 +167,10 @@ void setBankNormal(const V3D &pos1, const V3D &pos2, const V3D &basePos) {
   auto vec1 = pos1 - basePos;
   auto vec2 = pos2 - basePos;
   auto normal = vec1.cross_prod(vec2);
-  normal.normalize();
+  try {
+    normal.normalize();
+  } catch (std::runtime_error &) {
+  }
```

Comment:
A better solution would be to change line 222 of this file to
```
setBankNormal(corners.bottomRight() + bottomRightOffset, corners.topLeft() + topLeftOffset, basePos);
```
This would fix the crash and set the normal correctly for these instruments.
The catch clause here should log a warning.